### PR TITLE
Image and brief bio

### PR DESCRIPTION
**Pull request by MattsonCam**
I just added my headshot and a brief bio

### CODE REVIEW

**Comment by MattsonCam** [comment]
Hey @gwaybio , just a review reminder. Thanks!

File: _members/cameron-mattson.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
+---
+name: Cameron Mattson
+image: images/team/cameron_mattson.jpg
+description: Data Scientist
+role: BioData Scientist
```
Comment (gwaybio) [inline _members/cameron-mattson.md:5]: please make your role `programmer` (see https://github.com/WayScience/waysciencelab.com/blob/d005c611089ebbf5d8c245af1e72ccfa5299045c/_data/roles.yaml#LL17C6-L17C6)

File: _members/cameron-mattson.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+  - Cameron W. Mattson
+  - C. Mattson
+  - cmattson
+  - mcameron
+links:
+  email: cameron.mattson@cuanschutz.edu
+  github: https://github.com/MattsonCam
+---
+
+Cameron graduated with his B.S. in Bioengineering at the University of Colorado, where he implemented and developed physiological models. In his capstone project, he collaborated with a cardiovascular surgeon to design a medical device allowing access to the axilla during first rip resection surgeries. Cameron is currently pursuing his Graduate degree in Computer Science with a focus in Data Science and Data Engineering from the University of Colorado Boulder. He has worked on a variety of machine learning and data science project with both large and small companies.
```
Comment (gwaybio) [inline _members/cameron-mattson.md:17]: Please move to a [1 sentence per line format](https://sive.rs/1s) for markdown.

I think I also see a typo rip --> rib?

**Review by gwaybio — CHANGES_REQUESTED** [review]

File: _members/cameron-mattson.md (hunk 1 above)
Comment (MattsonCam) [reply _members/cameron-mattson.md]: Just added the change

File: _members/cameron-mattson.md (hunk 2 above)
Comment (MattsonCam) [reply _members/cameron-mattson.md]: Good catch!

**Comment by MattsonCam** [comment]
Hey @gwaybio , just sent a pr re-review request

File: _members/cameron-mattson.md [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+  - Cameron W. Mattson
+  - C. Mattson
+  - cmattson
+  - mcameron
+links:
+  email: cameron.mattson@cuanschutz.edu
+  github: https://github.com/MattsonCam
+---
+
+Cameron graduated with his B.S. in Bioengineering at the University of Colorado, where he implemented and developed physiological models.<br>
```
Comment (gwaybio) [inline _members/cameron-mattson.md:17]: please remove the line breaks (`<br>`)

It makes the deploy look a bit wonky.

I'll merge after! thanks!

Comment (MattsonCam) [reply _members/cameron-mattson.md]: Sounds good, changed!

**Comment by MattsonCam** [comment]
Just finished making the changes @gwaybio

**Review by gwaybio — APPROVED** [review]
Looks great! 🤩

**Pull request merged**